packages/calypso-build/webpack/file-loader.js

```diff
@@ -7,7 +7,7 @@ module.exports.loader = () => ( {
 	test: /\.(?:gif|jpg|jpeg|png|svg)$/i,
 	use: [
 		{
-			loader: 'file-loader',
+			loader: require.resolve( 'file-loader' ),
```

Comment:
Why did this line need to be changed?

Reply:
It didn't strictly need to, but using `require.resolve()` rather than just referencing the loader's name as a string literal is a mechanism to detect a missing loader dependency early (which we've been using in pretty much all the other shards). Arguably some sort of guard against accidentally dropping the `file-loader` dep in `package.json` (however unlikely).